pycon-us-2017/videos/dave-forgac-share-your-code-python-packaging-without-complication-pycon-2017.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "If you want people to use your code you should package it! You may have\nheard that packaging is hard but the Python packaging ecosystem has\nevolved a lot over the years. Taking your beautiful code and sharing it\nwith the world is complex but it doesn't have to be complicated. In this\ntalk you will learn how to take advantage of modern tooling and\npractices so you can get boring stuff out of the way, publish quickly\nand frequently, and focus on your code.\n\nThis talk will cover:\n\n-  A (brief) history of Python packaging\n-  Python Packaging User Guide recommendations\n-  Distribution formats\n-  Anatomy of a package\n-  Automating package creation\n-  Adding:\n-  Testing\n-  CI\n-  Documentation\n-  Testing package installation\n-  Releasing to PyPI\n\nThis talk is for you if you're new to Python packaging and would like to\nlearn how to share you code or if you've worked with Python for a while\nand just aren't up-to-date with the latest packaging practices.\n",
```

Comment:
Missing indentation:
~~~ json
  "description": "If you want people to use your code you should package it! You may have\nheard that packaging is hard but the Python packaging ecosystem has\nevolved a lot over the years. Taking your beautiful code and sharing it\nwith the world is complex but it doesn't have to be complicated. In this\ntalk you will learn how to take advantage of modern tooling and\npractices so you can get boring stuff out of the way, publish quickly\nand frequently, and focus on your code.\n\nThis talk will cover:\n\n-  A (brief) history of Python packaging\n-  Python Packaging User Guide recommendations\n-  Distribution formats\n-  Anatomy of a package\n-  Automating package creation\n-  Adding:\n    -  Testing\n    -  CI\n    -  Documentation\n-  Testing package installation\n-  Releasing to PyPI\n\nThis talk is for you if you're new to Python packaging and would like to\nlearn how to share you code or if you've worked with Python for a while\nand just aren't up-to-date with the latest packaging practices.\n",
~~~